src/traces/sankey/index.js

```diff
@@ -0,0 +1,31 @@
+/**
+* Copyright 2012-2017, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+var Plot = {};
+
+Plot.attributes = require('./attributes');
+Plot.supplyDefaults = require('./defaults');
+Plot.calc = require('./calc');
+Plot.plot = require('./plot');
+Plot.colorbar = require('./colorbar');
+
+Plot.moduleType = 'trace';
+Plot.name = 'sankey';
+Plot.basePlotModule = require('./base_plot');
+Plot.categories = ['gl'];
```

Comment:
Thanks for the catch, `gl` was a leftover, I'll still need to eyeball the diffs as the PR ripens. I did 
```
Plot.categories = [];
```
with which everything worked fine. I had to specify _some_ set (even if the empty set) otherwise `registry.js` throws. I also tried
```
Plot.categories = ['sankey'];
```
but it failed on some coercion.